frotz/src/dumb/dumb_output.c

```diff
@@ -522,6 +522,7 @@ void dumb_init_output(void)
 {
     if (h_version == V3) {
 	h_config |= CONFIG_SPLITSCREEN;
+	h_config |= CONFIG_NOSTATUSLINE;
```

Comment:
Using the option to disable status line for V3 games. This simplifies the `clean_observation_...` functions.